.codeclimate.yml

```diff
@@ -5,6 +5,9 @@ engines:
   pmd:
     enabled: true
     channel: beta
+  codenarc:
+    enabled: true
+    channel: beta
```

Comment:
That was quick! This hasn't been released yet though. Will comment here when I have an update.

Reply:
I'm surprised that this build didn't error, provided the unknown engine... @wfleming?

Reply:
Engine beta is out 👍

Reply:
I was surprised it didn't fail too. I've seen analysis failing when enabling a not released engine before. Didn't make sense to me why it didn't say anything at that time.